don't try to use the local version of GL headers

--- a/gl_util.h
+++ b/gl_util.h
@@ -13,12 +13,7 @@
 #endif
 
 /* Provide GL header files for Windows */
-#define INCLUDE_LOCAL_HEADER !defined(HAVE_W32API)
-#if INCLUDE_LOCAL_HEADER
-#include "./include/GL/gl.h"
-#else
 #include <GL/gl.h>
-#endif
 
 /* Use version-detection if available */
 #if defined(HAVE_VER)
